### PR TITLE
Transportation View

### LINKED ISSUE #15: Transportation
> This will just be a list of options, with it calculating and displaying time until the next bus (when the busses are running)

### PR DESCRIPTION
**Pull request by elijahverdoorn**
It's a little ugly, but it's done enough for version 1. This closes #15 (for now).

### CODE REVIEW

**Comment by hawkrives** [comment]
Don't you just love moment.js?

Anyway. LGTM. :shipit: If something's broken after the merge, I'll fix it tonight.

**Comment by hawkrives** [comment]
Oh no. I summoned the travis demons.

**Comment by hawkrives** [comment]
Eh. It's just eslint. I'll fix it and push later. We'll merge this.

You went significantly further than I was imagining, though :smile: I was just thinking at the v1 level… this is cool.

**Comment by elijahverdoorn** [comment]
I might have gotten carried away... It was fun.

File: views/transportation/tabs.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import BusView from './bus'
+// import PauseMenuView from './pause'
+
+export default [
+  {
+    id: 'bus',
+    title: 'Bus',
+    rnVectorIcon: {
+      iconName: 'bus',
+      selectedIconName: 'bus',
```
Comment (hawkrives) [inline views/transportation/tabs.js:14]: For future reference: these are the icons you can choose from. http://www.entypo.com/

`bus` is not one of them 😝

File: views/transportation/tabs.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    },
+    content: BusView,
+  },
+  {
+    id: 'otherModes',
+    title: 'Other Modes',
+    rnVectorIcon: {iconName: 'cab'},
```
Comment (hawkrives) [inline views/transportation/tabs.js:21]: Nor is `cab`.

File: views/transportation/tabs.js (hunk 1 above)
Comment (elijahverdoorn) [inline views/transportation/tabs.js:14]: I assumed that we had all of these: https://github.com/oblador/react-native-vector-icons
Noted.

Comment (hawkrives) [reply views/transportation/tabs.js:14]: Sadly, it makes us pick one font package to import from.

**Pull request merged**